app/js/components/manageApps/ManageApps.jsx

```diff
@@ -76,19 +80,92 @@ export default class ManageApps extends React.Component {
     });
   }
 
+  handleAddonUploadModal(addonName, action, response) {
+    swal({
+      title: `Are you sure you want to ${action} ${addonName}`,
+      icon: "warning",
+      buttons: true,
+      dangerMode: true,
+    })
+      .then((performAction) => {
+        if (performAction) {
+          swal(`${addonName} addon is ${response}`, {
+            icon: 'success', 
+          });
+          this.handleUploadRequest();
+        } else {
+          this.handleClear();
+        }
+      });
+  }
+
   handleUpload() {
+    let zipedAddon = document.getElementById('fileInput').files[0];
```

Comment:
nitpick: shouldn't the `zipedAddon` variable be declared as `const` since it's not supposed to be mutated.